tests/integration/workflow/test_test_run.py

```diff
@@ -343,16 +343,22 @@ def compute_test_case_metrics(
 def test__test__function_evaluator(
     dummy_model: Model,
     dummy_test_suites: List[TestSuite],
-    dummy_test_samples: List[DummyTestSample],
 ) -> None:
     test(dummy_model, dummy_test_suites[0], dummy_evaluator_function)
     TestRun(dummy_model, dummy_test_suites[0], dummy_evaluator_function)
 
 
+def test__test__noop_evaluator(
+    dummy_model: Model,
+    dummy_test_suites: List[TestSuite],
+) -> None:
+    test(dummy_model, dummy_test_suites[0], noop_evaluator)
+    TestRun(dummy_model, dummy_test_suites[0], noop_evaluator)
```

Comment:
Does creating this `TestRun` object do anything?

Reply:
not sure -- copied from the above tests, but these tests seem more along the lines of "do we run into an error on construction" more so than "does the server do the right thing". Would need to do more testing if we do want to go in the direction suggested by this PR.

Reply:
I confirmed through an E2E test that the no-op is working as expected. Will expand on this test

Reply:
@consoull The right place for specific functionality tests might be on the server side. Tests within `kolena` are meant to ensure that these flows don't trigger errors in communication, while the server side integration tests ensure that data state is correct.